### PR TITLE
CTRSTestCase: Simplify expectedErrors set

### PR DESCRIPTION
**Pull request by Turbo87**

### CODE REVIEW

File: test/org/rust/lang/core/parser/RustCTRSTestCase.kt [hunk 1]
```diff
@@ -6,16 +6,16 @@ import com.intellij.psi.impl.DebugUtil
...
     fun testCtrs() {
         var nFilesVisited = 0
-        FileUtil.visitFiles(File(myFullDataPath, "test"), {
+        FileUtil.visitFiles(File(myFullDataPath), {
             if (it.isFile && it.extension == myFileExt.trimStart('.')) {
                 nFilesVisited++;
                 val text = FileUtil.loadFile(it, CharsetToolkit.UTF8)
                 val psi = createPsiFile(it.name, text)
-                val expectedError = expectedErrors.contains(it.path)
+                val expectedError = !expectedErrors.find { path -> it.path.endsWith(path) }.isNullOrEmpty()
```
Comment (matklad) [inline test/org/rust/lang/core/parser/RustCTRSTestCase.kt]: Could this be simplified to `expectedErrors.any { path -> it.path.endsWith(path) }` ?

I was wrong when I used `it` in this huge lambda :) May be a nice idea to change `it` to `file` (wow, there is even an intention for it).

Comment (Turbo87) [inline test/org/rust/lang/core/parser/RustCTRSTestCase.kt]: done

**Comment by matklad** [comment]
Lets squash it: both commits simplify CTRS.

**Comment by Turbo87** [comment]
done

**Comment by matklad** [comment]
Thanks very much for this! I've moved `ctrs` twice recently, and I really appreciate the change =)

**Pull request merged**